tools/dgeni/common/decorators.ts

```diff
@@ -42,6 +42,12 @@ export function isDeprecatedDoc(doc: any) {
   return (doc.tags && doc.tags.tags || []).some((tag: any) => tag.tagName === 'deprecated');
 }
 
+/** Whether the given document is annotated with the "@docs-primary-module" jsdoc tag. */
+export function isPrimaryModuleDoc(doc: any) {
+  return (doc.tags && doc.tags.tags || [])
+      .some((tag: any) => tag.tagName === 'docs-primary-module');
```

Comment:
Maybe in another PR, but we should have some docs about our custom JsDoc tags

Reply:
Yeah totally. I had the same thought when creating that PR and created https://github.com/angular/components/issues/16469 for tracking.